egs/chime4/s5_1ch/local/run_beamform_blstm_gev_6ch_track.sh

```diff
@@ -67,8 +67,14 @@ else
 fi
 
 mkdir -p $odir
-nIsolatedDirs=`ls local/nn-gev/data/audio/16kHz/isolated/ 2>/dev/null | wc -l`
-nIsolatedExtDirs=`ls local/nn-gev/data/audio/16kHz/isolated_ext/ 2>/dev/null | wc -l`
+
+if [[ -d local/nn-gev/data/audio/16kHz/isolated && -d local/nn-gev/data/audio/16kHz/isolated_ext ]];then
```

Comment:
I think you could get this more simply by doing`set +e` before those two commands and `set -e` right afterwards.

Reply:
Thanks for your suggestion Dr. Povey. I have changed it and it works.